apps/coscrad-frontend/src/components/resources/vocabulary-lists/vocabulary-list-select.tsx

```diff
@@ -0,0 +1,40 @@
+import { IFormField, IValueAndDisplay } from '@coscrad/api-interfaces';
+import { InputLabel, MenuItem, Select } from '@mui/material';
+import { useState } from 'react';
+
+const NO_SELECTION_PLACEHOLDER = '-';
```

Comment:
Should this be "Select" instead of "-"?

Reply:
Yea, good idea. Something like that would be good. Maybe "-select-" or "<select>" or something that indicates that it's not literally the string value "select".